Description: Install Files to package using makefile
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 ease-framework (1.16-1) UNRELEASED; urgency=medium
 .
   * Package build fixed
   * Dolar fix
   * Debian Paths to jquery.js fixed
   * Locale fixed
   * Ubuntu 18.04 compatible
   * Locale fixed
   * Locale fixed
   * Locale fixed
   * Locale fixed
   * Merge origin/master  Conflicts:  debian/composer.json
Author: Vítězslav Dvořák <info@vitexsoftware.cz>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2018-11-18

--- ease-framework-1.16.orig/Makefile
+++ ease-framework-1.16/Makefile
@@ -1,5 +1,5 @@
 DESTDIR ?= debian/ease-framework/DEBIAN
-libdir  ?= /usr/share/php/EaseFramework
+libdir  ?= /usr/share/php/Ease
 docdir  ?= /doc/ease-framework/html
 
 all: build install
@@ -14,6 +14,7 @@ fresh:
 install:
 	mkdir -p $(DESTDIR)$(libdir)
 	cp -r src/Ease/ $(DESTDIR)$(libdir)
+	cp -r debian/composer.json $(DESTDIR)$(libdir)
 	mkdir -p $(DESTDIR)$(docdir)
 	cp -r docs $(DESTDIR)$(docdir)
 	
